plugins/optimization-detective/detect.js

```diff
@@ -434,7 +434,8 @@ export default async function detect( {
 		);
 		if (
 			! isNaN( previousVisitTime ) &&
-			( getCurrentTime() - previousVisitTime ) / 1000 < freshnessTTL
+			( freshnessTTL < 0 ||
```

Comment:
A key bit I realized was missing. Without this, URL Metrics will always attempt to be submitted again even.